saleor/graphql/core/mutations.py

```diff
@@ -575,13 +575,30 @@ def perform_mutation(cls, _root, info, **data):
         instance = cls.get_instance(info, **data)
         data = data.get("input")
         cleaned_input = cls.clean_input(info, instance, data)
+        metadata_list = cleaned_input.pop("metadata", None)
```

Comment:
Maybe it is already covered but shouldn't we validate the permission of writing to metadata here?